src/i18n/locales/en-US.json

```diff
@@ -230,7 +230,7 @@
   "components.walletinit.createwallet.createwalletscreen.title": "Create a new wallet",
   "components.walletinit.createwallet.mnemonicbackupimportancemodal.confirmationButton": "I understand",
   "components.walletinit.createwallet.mnemonicbackupimportancemodal.keysStorageCheckbox": "I understand that my secret keys are held securely on this device only, not on the company`s servers",
-  "components.walletinit.createwallet.mnemonicbackupimportancemodal.newDeviceRecoveryCheckbox": "I understand that if this application is moved to another device or deleted, my money can be only recovered with the backup phrase that I have written down and saved in a secure place.",
```

Comment:
Avoid the word "money" when possible since there is a lot of debate about whether or not cryptocurrencies are "money" with legal implications